lib/compiler.js

```diff
@@ -1,9 +1,13 @@
 var hasOwn = Object.prototype.hasOwnProperty;
 var assert = require("assert");
-var types = require("ast-types");
+var types = require("ast-types/fork")([
+  require("ast-types/def/esprima"),
+  require("ast-types/def/babel6")
+]);
```

Comment:
I think it's OK for now as the main weight comes from acorn. Let's move ast-types talk to https://github.com/benjamn/ast-types/pull/209.